### ISSUE TITLE
nan 

### ISSUE THREAD

**Issue by ImprovementMajor** [ImprovementMajor/NewNew#8]
@bodrovis Hi Ilya! Can you please take a look at this code? I tried to show exact lines by creating issue inside of the code,  but looks like it didnt work out. Thank you 
https://github.com/ImprovementMajor/NewNew/commit/9a7dbcc29092270f05e5713e4cb46902f80545ef

Here is the screenshot of submitting it as an issue as well, why it didn't work? Thank you
<img width="1188" alt="Screen Shot 2021-01-20 at 11 25 54 AM" src="https://user-images.githubusercontent.com/71445477/105204565-b2aea780-5b1a-11eb-8c8a-6529bf73b903.png">

**Comment by bodrovis** [comment]
Good day! Strange, I believe there was an issue with GitHub recently so I did not receive a notification about this comment. Sorry! So, speaking about numbers checking... In Python we don't have NaN (as this is a JS term) but there's something we can do. The simplest thing is to use `isdigit()` method:

```python
'1'.isdigit() # => True
'a'.isdigit() # => False
```

However, if you have not yet proceeded to the "Methods" module, then I'd recommend putting methods aside for now and simply check that the user has entered *something* at least. Also, in the upcoming module we'll discuss exceptions which will allow us to make programs more bulletproof (because, if the user enters some random gibberish then converting it to integer or float will result in an error).

Also, exceptions will probably work best if the user can theoretically enter a float number like `4.2`. Problem is, there's no built-in method that can safely check floats (which is actually really annoying):

```python
"4.2".isdigit() # => False, though it's clearly a number
```

As a bottomline, for now you can employ `isdigit()` or wait until the further modules to cover methods and exceptions in more detail.

P.S. There's yet another approach: regular expressions but these are a bit more advanced

**Comment by ImprovementMajor** [comment]
thank you! I thought NaN is in all languages lol

**Comment by bodrovis** [comment]
Nope, different languages have different approaches! In Python we have `None` but it simply means "no value at all"
